apps/emqx_bridge_gcp_pubsub/test/emqx_bridge_gcp_pubsub_producer_SUITE.erl

```diff
@@ -310,7 +310,7 @@ gcp_pubsub_config(Config) ->
         io_lib:format(
             "bridges.gcp_pubsub.~s {\n"
             "  enable = true\n"
-            "  connect_timeout = 1s\n"
+            "  connect_timeout = 5s\n"
```

Comment:
this is needed for less flakiness.

Reply:
I think it might have broken `t_get_status_timeout_calling_workers`: https://github.com/emqx/emqx/pull/12222